File: app/src/main/java/com/van/shortcuts/TwoActivity.java
package com.van.shortcuts;

import android.app.Activity;
import android.os.Bundle;
import android.util.Log;

public class TwoActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_two);
        Log.i(MainActivity.TAG, getIntent().getAction());
    }
}

File: app/src/main/java/com/van/shortcuts/TwoActivity.kt
package com.van.shortcuts

import android.app.Activity
import android.os.Bundle
import android.util.Log

class TwoActivity : Activity() {

    companion object {
        private const val TAG = "TwoActivity"
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_two)
        Log.i(TAG, intent.action)
    }
}
